macros/supporting/hash/hash.sql

```diff
@@ -0,0 +1,31 @@
+{# Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+#}
+
+
+{%- macro hash(columns=none, alias=none, is_hashdiff=false, hash_algo=none) -%}
```

Comment:
I renamed `hashdiff` to `is_hashdiff`. Generally it would be great to prefix bools with `is` or `has` as to not confuse them with something else (like providing a hashdiff, as unlikely as that is)

Reply:
Yes, good point.